src/main/scala/org/broadinstitute/dsde/firecloud/model/ModelJsonProtocol.scala

```diff
@@ -138,14 +138,14 @@ object ModelJsonProtocol {
   implicit object impQueryMap extends JsonFormat[QueryMap] {
     override def write(inputmap: QueryMap): JsValue = inputmap match {
       case matchall : ESMatchAll => matchall.toJson
-      case wildcard : ESWildcard => wildcard.toJson
+      case wildcard : ESMatch => wildcard.toJson
```

Comment:
nit: change `case widcard` to a more appropriate variable name? (but not "match" because that's a Scala word)